packages/api-client/index.ts

```diff
@@ -94,6 +94,23 @@ interface SearchClass {
   coreqs?: INEUAndReq | INEUOrReq;
   maxCredits: number;
   minCredits: number;
+  termId: string;
+}
+
+function occurrencesToCourseByCatalogYear(
+  occurrences: SearchClass[],
+  catalogYear: number
+): ScheduleCourse2<null> {
+  for (const occurrence of occurrences) {
+    const year = occurrence.termId.slice(0, 4);
+
+    if (year + 1 === catalogYear.toString()) {
```

Comment:
adjust by 1 here bc our catalog year refers to the first part of the catalog year range (i.e. 2019 for 2019-2020)